### ISSUE TITLE
Links in markup do not work

### ISSUE THREAD

**Issue by ralfstx** [eclipsesource/tabris-js#286]
Example: 

``` js
tabris.create("TextView", {
  markupEnabled: true,
  text: '<a href="http://tabrisjs.com">link</a>'
});
```

The link is displayed but tapping on it does nothing.

**Comment by jordiboehmelopez** [comment]
The support for links in markup text is only to visualize/format the text as link.
There is no support for events and alike.

**Comment by ralfstx** [comment]
I think we should either remove support for `<a>` tags or provide the developer with information required to react on clicked links programmatically. At the moment, there is only a touch event on the entire TextView, regardless of whether the link or any other text has been tapped on. When the markup contains multiple links, they cannot be distinguished in a listener.

**Comment by mpost** [comment]
Yeah it is only a visual indicator right now. Do limit confusion i would also vote to remove it from the list of supported tags.

**Comment by jonek** [comment]
Why can't we just implement a default behavior: open the link in the mobile browser when clicked?

**Comment by jordiboehmelopez** [comment]
@jonek I already discussed this for an hour with you (a few months ago)... this is not possible on iOS.

**Comment by jonek** [comment]
Sorry, Jordi, if I can't remember every details of the discussion we had some month ago. I just tried the following: enter "http://heise.de" in the official Apple Notes app. Press enter. It gets rendered as a clickable link. Click on the link. The URL is opened in Safari.

I asked for the reasons why we can not implement a default behavior.

What would be the technical difference between the behavior of the Notes app and an imaginary default behavior in tabris.js?

**Comment by ralfstx** [comment]
Applications may want to react on links on their own. This would be counter to a default behavior.

**Comment by mpost** [comment]
Fixed in 2.6 with new `tapLink` event.
